tests/versioning_test.go

```diff
@@ -1075,7 +1075,7 @@ func (s *versioningIntegSuite) dispatchQuery() {
 	v11 := s.prefixed("v11")
 	v2 := s.prefixed("v2")
 
-	started := make(chan struct{}, 2)
+	started := make(chan struct{}, 10)
```

Comment:
Should this be a lower number to ensure max replays?

Reply:
It could but that's not really what's being tested here. I'd rather just leave the channel for synchronization.

We clear sticky on close so we expect it to replay.